src/main/java/io/wisetime/connector/api_client/support/RestRequestExecutor.java

```diff
@@ -115,7 +115,7 @@ String mergeNamedParams(String actionPath, List<NameValuePair> namedParamList) {
 
       // replace matched group with value
       mergedTemplate = new StringBuilder(mergedTemplate)
-          .replace(startIndex, endIndex, URIUtil.encodePath(keyValuePair.getValue()))
+          .replace(startIndex, endIndex, UrlEscapers.urlPathSegmentEscaper().escape(keyValuePair.getValue()))
```

Comment:
Root cause for the issue is that some IRN has slash so we need this url escaper for path params that escapes slash (/)